docs/endpoints/endpoints.rst

```diff
@@ -0,0 +1,743 @@
+:ref:`client credentials with globus compute clients`.
+
+
+.. _enable_on_boot:
+
+Starting the Compute Endpoint on Host Boot
+------------------------------------------
+
+Run ``globus-compute-endpoint enable-on-boot`` to install a systemd unit file:
+
+.. code-block:: console
+
+   $ globus-compute-endpoint enable-on-boot my_first_endpointendpoint
+   Systemd service installed. Run
+      sudo systemctl enable globus-compute-endpoint-my_first_endpoint.service --now
+   to enable the service and start the endpoint.
+
+Run ``globus-compute-endpoint disable-on-boot`` for commands to disable and uninstall
+the service:
+
+.. code-block:: console
+
+   $ globus-compute-endpoint disable-on-boot my-endpoint
+   Run the following to disable on-boot-persistence:
+      systemctl stop globus-compute-endpoint-my-endpoint
+      systemctl disable globus-compute-endpoint-my-endpoint
+      rm /etc/systemd/system/globus-compute-endpoint-my-endpoint.service
+
+
+AMQP Port
+---------
+
+Endpoints receive tasks and communicate task results via the AMQP messaging protocol.
+As of v2.11.0, newly configured endpoints use AMQP over port 443 by default, since
+firewall rules usually leave that port open. In case 443 is not open on a particular
+cluster, the port to use can be changed in the endpoint config via the ``amqp_port``
+option, like so:
+
+.. code-block:: yaml
+
+   amqp_port: 5671
+   display_name: My Endpoint
+   engine: ...
+
+Note that only ports 5671, 5672, and 443 are supported with the Compute hosted services.
+Also note that when ``amqp_port`` is omitted from the config, the port is based on the
+connection URL the endpoint receives after registering itself with the services, which
+typically means port 5671.
+
+.. _endpoints_templating_configuration:
+
+Templating Endpoint Configuration
+---------------------------------
+
+A common experience for Compute users is a proliferation of their Compute Endpoints.
+After starting with a basic configuration, changing or conflicting requirements
+necessitate running multiple endpoints |nbsp| --- |nbsp| sometimes simultaneously |nbsp|
+--- |nbsp| with different configurations.  For example, attributing work to different
+accounts, changing the size of the provisioned compute resource to match the demands of
+the problem, or changing the submission queue to the batch‑system.
+
+This becomes a bit of an administrative mess for these users, who must constantly update
+their endpoint configurations, bring up and bring down different endpoints, and be aware
+of which endpoints have which configuration.
+
+As of May, 2024, Compute Endpoints may now be run as "multi‑user" endpoints.  Please
+ignore the name (see the note, below) and instead think of it as "template‑able".  A
+multi‑user endpoint specifies a configuration *template* that will be filled in by
+SDK‑supplied user‑variables.  This configuration is then applied to sub‑processes of the
```

Comment:
```suggestion
SDK‑supplied user‑variables.  This configuration is then applied to sub‑processes of the endpoint.
```

Reply:
I think this is already ... correct as is?